core/src/it/java/software/amazon/awssdk/core/http/ConnectionPoolMaxConnectionsIntegrationTest.java

```diff
@@ -57,12 +57,9 @@ public void leasing_a_new_connection_fails_with_connection_pool_timeout() throws
         AmazonSyncHttpClient httpClient = HttpTestUtils.testClientBuilder()
                                                        .clientExecutionTimeout(null)
                                                        .retryPolicy(RetryPolicy.NONE)
-                                                       .httpClient(ApacheSdkHttpClientFactory.builder()
-                                                                                         .connectionTimeout(
-                                                                                                 Duration.ofMillis(100))
-                                                                                         .maxConnections(1)
-                                                                                         .build()
-                                                                                         .createHttpClient())
+                                                       .httpClient(ApacheHttpClient.builder()
+                                                                               .connectionTimeout(Duration.ofMillis(100))
```

Comment:
Does this test still work? It seems maxConnections would be required.

Reply:
Whoops! Got a little over-zealous with the delete key on this one.